### DISCUSSION TITLE
Dynamic Options in qdate

### DISCUSSION THREAD

**Discussion by mila76** [quasarframework/quasar#10203] [General - Components / Directives / etc] [answered]
There is a way to dynamic change the options (clickable date) in qdate?
I cant find any methods on qdate that force the rerender of the calendar....

If i change options in runtime only when i switch manually month i can see the new options

I missing something or there is no way??
Thanks

**Comment by MilosPaunovic** [comment, marked as answer]
Please make a codepen that can reproduce the problem.

**Reply by mila76** [reply]
Creating codepen i see my own error....

Work as expected now
